src/wasm-interpreter.h

```diff
@@ -1174,28 +1242,171 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     assert(flow.values.size() > curr->index);
     return Flow(flow.values[curr->index]);
   }
+  Flow visitLocalGet(LocalGet* curr) {
+    NOTE_ENTER("LocalGet");
+    NOTE_EVAL1(curr->index);
+    // Check if a constant value has been set in the context of this runner.
+    auto iter = localValues.find(curr->index);
+    if (iter != localValues.end()) {
+      return Flow(std::move(iter->second));
```

Comment:
My understanding here is that creating the `std::pair<const wasm::Index, wasm::Literals>` will make a copy of the `wasm::Literals` *value*, so using a `std::move` here hints that we can move that volatile copy instead of copying twice and dumping one. Perfectly possible that I don't actually know what I'm doing. Please advise :)